src/core/src/op/paged_attention.cpp

```diff
@@ -8,6 +8,88 @@
 #include "itt.hpp"
 #include "openvino/op/op.hpp"
 
+namespace {
+
+using namespace ov;
+using namespace ov::element;
```

Comment:
This using should be added where literal list of elements specified. when `input_check` is called.